pkg/epinio/components/application/AppConfiguration.vue

```diff
@@ -124,6 +131,10 @@ export default Vue.extend<Data, any, any, any>({
           // Filter out any we don't know about
           this.values.configurations = this.initialApplication.baseConfigurationsNames?.filter((cc: string) => this.configurations.find((c: any) => c.value === cc)) || [];
         }
+
+        if (this.isFromManifest) {
+          this.values.configurations = this.application.configuration.configurations.filter((c: string) => !c.match(/^(x[a-z0-9]*)-([^-]+)/)) || [];
```

Comment:
Regex initialisation can be expensive, especially in a for loop. This is used in three places so worth making a `const` at the top of the file